dbus-macros/README.md

```diff
@@ -0,0 +1,4 @@
+Procedural macros for dbus crate
+--------------------------------
+
+Simplifies definition of complex dbus interfaces
```

Comment:
Umm, might need someting more here... And the main readme that lists all the crates in the workspace could advertise this crate as well.

Reply:
I'll try to force myself to write some actually readable docs, but no promises =)